syft/frameworks/torch/functions.py

```diff
@@ -1,7 +1,8 @@
 from .tensors.interpreters import MultiPointerTensor
+from syft.workers import VirtualWorker
 
 
-def combine_pointers(*pointers):
+def combine_pointers(*pointers: VirtualWorker) -> MultiPointerTensor:
```

Comment:
pointers should be a list of pointers not a virtual worker

Reply:
What is it to be given as type then?